tests/modules/dshbio/splitbed/nextflow.config

```diff
@@ -3,7 +3,7 @@ process {
     publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
 
     withName: DSHBIO_SPLITBED {
-        ext.suffix = '.'
+        ext.prefix = { "${meta.id}." }
```

Comment:
Maybe this should be kept just as `.`? I gues it's specifying an ourput directory?

Reply:
See above.